src/components/rmrk/Create/CreateItem.vue

```diff
@@ -17,7 +17,7 @@
       ></b-input>
       <Tooltip iconsize="is-medium" :label="$i18n.t('tooltip.name')" />
     </b-field>
-    <b-field :label="$i18n.t('Collection description')" class="mb-0">
+    <b-field :label="$i18n.t('NFT description')" class="mb-0">
```

Comment:
Wow, you have eyes of the 🦅 . 

The best fix:
- put in `en.json` -> `nft` -> `description`
- in this vue file then `$t('nft.description')`